javascript-forms/form-values.js

```diff
@@ -0,0 +1,17 @@
+const $formTwo = document.forms[0];
+const $formName = document.forms[0].name;
+const $formEmail = document.forms[0].email;
+const $formTextArea = document.forms[0].message;
+
+const handleSubmit = event => {
+  event.preventDefault();
+  const submitInfo = {
+    name: $formName.value,
+    email: $formEmail.value,
+    textArea: $formTextArea.value
+  };
```

Comment:
Rather than storing persistent references to the inputs (which can be a _very_ long list, and some inputs can share a `name` property), dynamically grab the values from the form's `elements` property during submission:

---

![image](https://user-images.githubusercontent.com/16143771/138144749-23abc687-204d-48e5-ae22-28f625919970.png)